lib/pure/options.nim

```diff
@@ -139,6 +266,12 @@ proc get*[T](self: Option[T], otherwise: T): T =
   else:
     otherwise
 
+template either*(self, otherwise: untyped): untyped =
```

Comment:
In many languages `Either` is another data type, related to Option. Using `either` here can be confusing, and it seems not necessary, since it is just an alias for `get`

Reply:
@krux02 asked if this would evaluate other if it was a procedure, and I think that it makes sense if it's rewritten to not do that and keep both either and get with default. This means you can do `either(<complex options statement>, procedureWithSideEffects()) and only if nothing comes of the options statement then will the side effects procedure be run.